src/VisualStudio/Core/Def/Implementation/Utilities/VsHierarchyExtensions.cs

```diff
@@ -51,8 +50,13 @@ public static string GetMonikerForHierarchyAndItemId(this IVsRunningDocumentTabl
                 }
             }
 
-            // Uh, OK, that's probably not good that we're supposedly an open file but not in the RDT. We'll fall back
-            // to the IVsHierarchy's name property for this item.
+            // Uh, OK, that's probably not good that we're supposedly an open file but not in the RDT.
+            return null;
+        }
+
+        // Gets the IVsHierarchy's name property for this item.
```

Comment:
Sorry, missed this. I didn't want to make one more commit as I'll be porting this to preview branch as well. But I have an idea to remove the RDT look up and just it with [`IVsProject.GetMkDocument`](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.shell.interop.ivsproject.getmkdocument.aspx). Haven't tested the idea yet, but just an idea at this point.